### PR TITLE
Refactored group_sliders method

### PR DESCRIPTION
**Pull request by FarmHJ**

### CODE REVIEW

File: seirmo/tests/test_slider.py [hunk 1]
```diff
@@ -15,6 +15,28 @@ class TestSliderComponent(unittest.TestCase):
...
+        # Check that 2 slider group Div is created
+        self.assertEqual(len(slider().children), 2)
+
+        # Check that there are 3 parts (structure) in each slider group # noqa
+        self.assertEqual(len(slider().children[0].children), 3)
+        self.assertEqual(len(slider().children[1].children), 3)
+
+        # Check that labels and slider is added for each slider group # noqa
+        self.assertEqual(len(slider().children[0].children[2].children), 4)
+        self.assertEqual(len(slider().children[1].children[2].children), 2)
```
Comment (DavAug) [inline seirmo/tests/test_slider.py:38]: that's exactly what I had in mind. Very good!

**Review by DavAug — CHANGES_REQUESTED** [review]
Really good job @FarmHJ ! I just made a comment about the assert that one slider can only belong to one group, and a couple style comments that may be helpful.

File: seirmo/apps/_slider.py [hunk 2]
```diff
@@ -21,6 +21,26 @@ def __init__(self): # noqa
         self._sliders = {}
         self._group_ids = {}
 
+    def __call__(self):
+
+        slider_group_component = []
+        for group_id in list(self._group_ids.keys()):
```
[the remark this answers is not in the document]
Comment (FarmHJ) [reply seirmo/apps/_slider.py]: If I understand it correctly, changing it this way would change the grouping of the sliders

[the remark this answers is not in the document]
Comment (FarmHJ) [reply seirmo/apps/_slider.py]: Sorry about that, now I understand what you are trying to say.

File: seirmo/apps/_slider.py [hunk 3]
```diff
@@ -19,15 +19,19 @@ def __init__(self): # noqa
...
+        """
 
         slider_group_component = []
-        for group_id in list(self._group_ids.keys()):
+
+        for group_id, slider_id in self._slider_groups.items():
 
             slider_object = []
-            for slider_members in self._group_ids[group_id]:
+            for slider_members in slider_id:
```
Comment (DavAug) [inline seirmo/apps/_slider.py:34]: sliders_ids? This is more than one slider isn't it?

Comment (DavAug) [reply seirmo/apps/_slider.py:34]: and then maybe slider_member, because this is only one slider

**Review by DavAug — APPROVED** [review]
Very nice @FarmHJ ! I just have one super minor comment on the variable names, see below.

**Pull request merged**